src/pybind/mgr/progress/module.py

```diff
@@ -368,7 +364,64 @@ def _osd_out(self, old_map, old_dump, new_map, osd_id):
         ev.pg_update(self.get("pg_dump"), self.log)
         self._events[ev.id] = ev
 
-    def _osd_in(self, osd_id):
+    def _osd_in(self, old_map, old_dump, new_map, osd_id):
```

Comment:
since this is almost the same as _osd_out(), perhaps they could be combined into a single method with a parameter for in/out